### PR TITLE
Finalizing the content and adding recommendations

### PR DESCRIPTION
**Pull request by bumi**
First round of recommendations

### CODE REVIEW

**Comment by bumi** [comment]
new version of #25

File: app/javascript/views/Question.vue [hunk 1]
```diff
@@ -15,7 +15,7 @@
             {{ question.title }}
           </h4>
           <p class="my-0 text-sm opacity-80">
-            {{ question.description }}
+            <span v-html="question.description_formatted"></span>
           </p>
         </div>
```
Comment (musayann) [inline app/javascript/views/Question.vue]: We should replace the paragraph with this 
```vue.js

<div class="my-0 text-sm opacity-80" v-html="question.description_formatted">
</div>
```

Comment (bumi) [reply app/javascript/views/Question.vue]: thanks!

**Comment by musayann** [comment]
I think we can merge this one

**Pull request merged**